rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -260,11 +263,114 @@ echo "Trust all proxies"
 anonaddy vendor:publish --no-interaction --provider="Fideloper\Proxy\TrustedProxyServiceProvider"
 sed -i "s|^    'proxies'.*|    'proxies' => '\*',|g" /var/www/anonaddy/config/trustedproxy.php
 
+##
+## RSPAMD
+##
+
+if [ "$RSPAMD_ENABLE" = "true" ]; then
```

Comment:
I think we should have a, additional check above:

```shell
if [[ "$RSPAMD_ENABLE" = "true" && ( "$DKIM_ENABLE" = "true" || "$DMARC_ENABLE" = "true" ) ]]; then
  # exit because not mutually exclusive
fi
```